miniogw/gateway.go

```diff
@@ -639,6 +639,12 @@ func minioObjectInfo(bucket, etag string, object *uplink.Object) minio.ObjectInf
 	if etag == "" {
 		etag = object.Custom["s3:etag"]
 	}
+	// Set Headers to properly return videofiles
+	if needsVideoHeaders(object.Key) {
```

Comment:
Why doesn't the original upload contain the correct headers?

I suspect we are missing a ton of video extensions.

Reply:
We are, this is more or less a quick fix as we do not allow metadata editing.